From c1dfa045a4f9c40f3e52df5a0aa40c53344fd918 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Sat, 4 Jan 2020 14:39:29 +0200
Subject: [PATCH 19/19] tests: Disable actor-shader-effect

Disable actor-shader-effect for test failures that upstream doesn't care
about either as per commit 7943a099f394863198fd6ae0cc5e873ea8e52a19
---
 tests/conform/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/conform/Makefile.am b/tests/conform/Makefile.am
index af1e64b93..e4ef20cfc 100644
--- a/tests/conform/Makefile.am
+++ b/tests/conform/Makefile.am
@@ -24,7 +24,6 @@ actor_tests = \
 	actor-offscreen-redirect \
 	actor-paint-opacity \
 	actor-pick \
-	actor-shader-effect \
 	actor-size \
 	$(NULL)
 
-- 
2.20.1

